src/__internal__/utils/helpers/browser/browser.js

```diff
@@ -188,6 +188,31 @@ const Browser = {
   submitForm: (form) => {
     form.submit();
   },
+
+  /**
+   * From an existing node, focus the next focusable element in the DOM.
+   *
+   * @param {Node} refNode
+   */
+  focusNextElement: (refNode) => {
+    const treeWalker = document.createTreeWalker(
+      Browser.getDocument().body,
+      NodeFilter.SHOW_ELEMENT,
+      {
+        acceptNode(node) {
+          return node.tabIndex >= 0 &&
+            // eslint-disable-next-line no-bitwise
```

Comment:
please, give a hint which option you'd like to use to avoid comment.
https://eslint.org/docs/rules/no-bitwise#options

Reply:
This approach is fine 👍🏻